test/new-e2e/tests/agent-shared-components/config-refresh/non_core_agents_sync_common_test.go

```diff
@@ -0,0 +1,97 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
```

Comment:
Should we add the test build tag?

Reply:
Hmm there is no build tag for any of the tests in `test/new-e2e` so I'm not sure it's worth adding a tag here 🤷

Reply:
Ohh sorry. I did not see that the file end in `_test.go` this would get the build tag automatically from Go 😄

Reply:
I think it is fine here as none of the code under `test/new-e2e` is used by the agent, with or without the `_test.go` suffix, no need to exclude it with build tags

Reply:
By the way @Kaderinho no need to use the `_test.go` suffix for this file since it does not contain any test.